tests/CellDivisionTests.cpp

```diff
@@ -2,20 +2,20 @@
 #include <Automaton.h>
 #include "StateHelper.h"
 
-TEST_CASE("CellDivision - basic test") {
+TEST_CASE("RepairCells & CellDivision - basic test") {
```

Comment:
Dlaczego połączyłeś te testy? (RepairCells i CellDivision)

Reply:
W obydwu funkcjach (i tylko tych) są generowane liczby losowe, więc przy rozdzielonych testach, nie umiałbym zachowywać odpowiedniego stanu generatora przy kolejnych iteracjach.

Reply:
Oh ok. Może warto by to było dopisać w komentarzu?